data/json/speech.json

```
@@ -2262,6 +2262,42 @@
     "sound": "Wanna play with me?",
     "volume": 10
   },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_sapien_spear" ],
+    "sound": "Save your tears.",
+    "volume": 10
+  },
+  {
+    "type": "speech",
+    "speaker": [ "mon_feral_sapien_spear" ],
+    "sound": "Soon enough, you'll become one with the blob!",
+    "volume": 10
```

```suggestion
    "volume": 15
```
Since there's an exclamation point.